### ISSUE TITLE
Database creation script error in MariaDB version >= 10.2.1

### ISSUE THREAD

**Issue by hannakoppelaar** [DIA-NZ/webcurator#114]
When you run the script setup-mysql.sh or execute the sql file wct-schema-mysql.sql directly inside the mysql client the following error occurs: 

Function or expression '`TO_DATE`()' cannot be used in the CHECK clause of `CHK_END_TIME`

This is probably an issue for MariaDB 10.2.1 and up, since from that version onwards constraints are actually enforced.

**Comment by jmvezic** [comment]
This is still a problem after one year, making it impossible to install WCT with the newest MariaDB version.

MariaDB version 10.3.22.

**Comment by hannakoppelaar** [comment]
@jmvezic Yeah, it's annoying. There's a fix in the coming release, which upgrades the major frameworks and libraries used in wct. It's in a new repository. This is the commit: https://github.com/WebCuratorTool/webcurator/pull/1/commits/8be81ac36f28a9120cdc72d28efd53068197c29f. 

As a workaround, for now, you could replace your version of wct-schema-mysql.sql with the version in that commit, i.e.: https://github.com/WebCuratorTool/webcurator/blob/8be81ac36f28a9120cdc72d28efd53068197c29f/webcurator-db/latest/sql/wct-schema-mysql.sql.

Alternatively, you could wait for the new release, which we're hoping to finish this summer.

**Comment by jmvezic** [comment]
Yes, I did exactly that - works perfectly. Does the coming release also cover the [OpenJDK 11 support](https://github.com/DIA-NZ/webcurator/issues/130) perhaps?

**Comment by hannakoppelaar** [comment]
@jmvezic We're not planning to have open-jdk 11 support in the first release, 3.0, but it will be in the first or second minor release after that. It does have a fairly high priority for us.
